src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -873,6 +873,33 @@
     (SSLUtils/caCertAndCrlPemsToSSLContext ca-cert-reader
                                                        crls-reader)))
 
+(defn generate-ssl-context
+  [options]
```

Comment:
While trying to port `generate-ssl-context` over to Java, it became clear to me that it doesn't make sense as a Java method. I talked it over with @nwolfe and we have decided to leave it as a clojure function.